src/mongocxx/test/spec/unified_tests/runner.cpp

```diff
@@ -73,6 +74,88 @@ entity::map& get_entity_map() {
     return init_maps().second;
 }
 
+const auto kLocalMasterKey =
+    "\x32\x78\x34\x34\x2b\x78\x64\x75\x54\x61\x42\x42\x6b\x59\x31\x36\x45\x72"
+    "\x35\x44\x75\x41\x44\x61\x67\x68\x76\x53\x34\x76\x77\x64\x6b\x67\x38\x74"
+    "\x70\x50\x70\x33\x74\x7a\x36\x67\x56\x30\x31\x41\x31\x43\x77\x62\x44\x39"
+    "\x69\x74\x51\x32\x48\x46\x44\x67\x50\x57\x4f\x70\x38\x65\x4d\x61\x43\x31"
+    "\x4f\x69\x37\x36\x36\x4a\x7a\x58\x5a\x42\x64\x42\x64\x62\x64\x4d\x75\x72"
+    "\x64\x6f\x6e\x4a\x31\x64";
+
+bsoncxx::document::value get_kms_values() {
+    char key_storage[96];
+    memcpy(&(key_storage[0]), kLocalMasterKey, 96);
+    bsoncxx::types::b_binary local_master_key{
+        bsoncxx::binary_sub_type::k_binary, 96, (const uint8_t*)&key_storage};
+
+    auto kms_doc = make_document(
+        kvp("aws",
+            make_document(
+                kvp("accessKeyId", test_util::getenv_or_fail("MONGOCXX_TEST_AWS_ACCESS_KEY_ID")),
+                kvp("secretAccessKey",
+                    test_util::getenv_or_fail("MONGOCXX_TEST_AWS_SECRET_ACCESS_KEY")))),
+        kvp("azure",
+            make_document(
+                kvp("tenantId", test_util::getenv_or_fail("MONGOCXX_TEST_AZURE_TENANT_ID")),
+                kvp("clientId", test_util::getenv_or_fail("MONGOCXX_TEST_AZURE_CLIENT_ID")),
+                kvp("clientSecret",
+                    test_util::getenv_or_fail("MONGOCXX_TEST_AZURE_CLIENT_SECRET")))),
+        kvp("gcp",
+            make_document(
+                kvp("email", test_util::getenv_or_fail("MONGOCXX_TEST_GCP_EMAIL")),
+                kvp("privateKey", test_util::getenv_or_fail("MONGOCXX_TEST_GCP_PRIVATEKEY")))),
+        kvp("kmip", make_document(kvp("endpoint", "localhost:5698"))),
+        kvp("local", make_document(kvp("key", local_master_key))));
+
+    return kms_doc;
+}
+
+bsoncxx::document::value parse_kms_doc(bsoncxx::document::view_or_value test_kms_doc) {
+    auto kms_values = get_kms_values();
```

Comment:
```suggestion
    const auto kms_values = get_kms_values();
```
Const-qualify local variables that are not modified within this function.